stack-exact.yaml

```
@@ -7,7 +7,7 @@
 # You can add --resolver xxx to the above command to have the effect
 # of overriding the choice of compiler if desired.
 
-resolver: ghc-9.2.4
+resolver: ghc-9.2.2
```

it seems stackage [lts-20.8](https://www.stackage.org/lts-20.8) supports ghc 9.2.5, so why not

```suggestion
resolver: ghc-9.2.5
```
?

---

i chose 9.2.2 because its the minimum supported build compiler for ghc-9.6.1 flavored builds.

---

gotcha! thanks for the explanation